site/layouts/_default/docs.html

```diff
@@ -62,4 +62,5 @@ <h1 class="bd-title mb-0" id="content">{{ .Title | markdownify }}</h1>
   {{ range .Page.Params.extra_js -}}
     <script{{ with .async }} async{{ end }} src="{{ .src }}"></script>
   {{- end -}}
+  <div class="position-fixed"><input type="text"></div>
```

Comment:
Sorry to disturb this merged PR, but this `div/input` looks useless to me. Is that a mistake or did misunderstood something in here ?

Reply:
This is a hack to keep Safari from jumping to the bottom of the page. It's a docsearch bug that hasn't been fixed yet.

Reply:
Oh ok, thanks a lot for your answer !